### DISCUSSION TITLE
[Support]: Frigate won't start or allow configuration

### DISCUSSION THREAD

**Discussion by zs3y** [blakeblackshear/frigate#14974] [General Support] [answered]
### Describe the problem you are having

New install of HAOS, Frigate installed via HACS, but will not configure under devices.
Frigate should be version 5.4.0 as per HACS page
![Screenshot from 2024-11-14 10-49-46](https://github.com/user-attachments/assets/f47dfde0-5fe5-4e52-80d9-1f48d6be86aa)
![Screenshot from 2024-11-14 10-47-47](https://github.com/user-attachments/assets/e320f4d4-afa3-4004-ba79-b3e280cef650)
![Screenshot from 2024-11-14 10-47-11](https://github.com/user-attachments/assets/7c19fe68-c3b8-450b-a4c0-4b3144935c09)

### Version

on the settings -> about page I get -  Core 2024.11.1 Supervisor 2024.11.2 Operating System 13.2 Frontend 20241106.2

### Frigate config file

```yaml
there's no Frigate config file as yet...
```

### Relevant Frigate log output

```shell
there's no Frigate log as yet...
```

### Relevant go2rtc log output

```shell
not available as yet
```

### FFprobe output from your camera

```shell
this won't help either...
```

### Frigate stats

```json
none availale
```

### Install method

HassOS Addon

### docker-compose file or Docker CLI command

```yaml
I do not know where to find this, this would have been automatically downloaded or generated by HACS, I presume...
```

### Object Detector

CPU (no coral)

### Network connection

Wired

### Camera make and model

some WhoFlungDung PoE camera, but not relevant as yet

**Comment by zs3y** [comment]
I'm pretty sure its not relevant, but the browser I failed to specify is Firefox 115.15.0esr (64-bit)

**Comment by hawkeye217** [comment, marked as answer]
Check out the [documentation](https://docs.frigate.video/frigate/installation). You need to install Frigate itself. The integration available through HACS is for connecting Home Assistant to an existing Frigate installation.

**Reply by zs3y** [reply]
Thanks dude, the RTFM should be default go to, I didn't realise that the HACS part was only the integration and not the actual application :-)

Thanks again, got it working now...

**Reply by hawkeye217** [reply]
Great, happy to help.
